### PR TITLE
Remove legacy UWP WebSocket module

### PR DESCRIPTION
**Pull request by JunielKatarn**
WebSocket implmementation has been shared at Shared\Modules\WebSocketModule.cpp for a long while.

This implementation should no longer be needed.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/7736)

### CODE REVIEW

**Comment by NickGerleman** [comment]
Change looks good to me. Just be aware that RNW 0.64 (which moved the websocket impl) is still being adopted by internal partners. So while the code has been merged for a while, we don't have much usage in the wild to use as feedback.

File: vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj [hunk 1]
```diff
@@ -380,93 +379,93 @@
...
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\components\view\ViewEventEmitter.cpp" DisableSpecificWarnings="4018;%(DisableSpecificWarnings)"/>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\components\view\ViewProps.cpp" DisableSpecificWarnings="4459;4715;%(DisableSpecificWarnings)"/>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\components\view\ViewShadowNode.cpp"/>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\components\view\YogaLayoutableShadowNode.cpp" DisableSpecificWarnings="4701;4715;%(DisableSpecificWarnings)"/>
-    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\components\view\YogaStylableProps.cpp" DisableSpecificWarnings="4715;%(DisableSpecificWarnings)"/>
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\attributedstring\AttributedStringBox.cpp" DisableSpecificWarnings="4715;%(DisableSpecificWarnings)" />
+    <ClCompile Include="$(ReactNativeDir)\ReactCommon\react\renderer\attributedstring\ParagraphAttributes.cpp" />
```
Comment (NickGerleman) [inline vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj:383]: Do you know how spacing changed for all of these? Did Visual Studio do this automatically?

File: vnext/Microsoft.ReactNative/Modules/CreateModules.cpp [hunk 2]
```diff
@@ -28,9 +27,7 @@ std::shared_ptr<IWebSocketResource> IWebSocketResource::Make(std::string &&urlSt
 
 std::unique_ptr<facebook::xplat::module::CxxModule> CreateWebSocketModule(
     Mso::CntPtr<Mso::React::IReactContext> &&context) noexcept {
```
Comment (NickGerleman) [inline vnext/Microsoft.ReactNative/Modules/CreateModules.cpp:29]: Can we remove the param for React context if no longer needed?

Consider removing this function entirely and just inlining the calls to make_unique if we don't need a factory function.

**Comment by NickGerleman** [comment]
There is code in OInstance around the loading of this module which is specialto alow UWP to load a different implementation from desktop. Since there is now only one implementation we can simply this.

**Review by dannyvv — APPROVED** [review]

**Review by jonthysell — APPROVED** [review]

File: vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj (hunk 1 above)
Comment (vmoroz) [reply vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj:383]: Visual Studio always puts a space before the '/>'. It seems that these lines were previously added manually and the VS formatted the XML after the items are removed in the VS IDE.

**Review by vmoroz — APPROVED** [review]
:shipit:

File: vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj (hunk 1 above)
Comment (JunielKatarn) [reply vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj:383]: Pretty much what Vladimir said.

**Comment by JunielKatarn** [comment]
> There is code in OInstance around the loading of this module which is special cased to alow UWP to load a different implementation from desktop. Since there is now only one implementation we can simply this.

UWP-specific conditions and includes have been removed.

Will merge tomorrow if no further feedback needs to be addressed.

**Comment by JunielKatarn** [comment]
> Change looks good to me. Just be aware that RNW 0.64 (which moved the websocket impl) is still being adopted by internal partners. So while the code has been merged for a while, we don't have much usage in the wild to use as feedback.

Agreed. However, the current WS module implementation does have very broad usage via Office/Desktop.

**Pull request merged**